src/panels/config/cloud/dialog-cloud-already-connected/dialog-cloud-already-connected.ts

```diff
@@ -62,6 +62,26 @@ class DialogCloudAlreadyConnected extends LitElement {
           </b>
         </div>
         <div class="instance-details">
+          ${details.name
+            ? html`<div class="instance-detail">
+                <span>
+                  ${this.hass.localize(
+                    "ui.panel.config.cloud.dialog_already_connected.instance_name"
+                  )}:
+                </span>
+                <span>${details.name})}</span>
```

Comment:
Missed this. The `)}` is extra here

Reply:
https://github.com/home-assistant/frontend/pull/24497